src/Tizen.NUI/src/public/WebView/WebContext.cs

```diff
@@ -529,7 +529,7 @@ public void RegisterUrlSchemesAsCorsEnabled(string[] schemes)
         /// Registers js plugin mime types.
```

Comment:
js -> JS

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520